src/getCoinCombination.test.js

```diff
@@ -7,4 +7,34 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+  
+  test('should return 0`s if original data is 0', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0])
```

Comment:
The test case is not descriptive enough. It should include more information about what the function is doing.